lib/ansible/modules/cloud/amazon/iam_role.py

```diff
@@ -47,6 +47,12 @@
         To embed an inline policy, use M(iam_policy). To remove existing policies, use an empty list item.
     required: true
     aliases: ['managed_policies']
+  remove_unlisted_policies:
```

Comment:
Can you name this "purge_policies" or "purge_unlisted_policies" instead to match the SG tags and ec2 modules?